jobs/JJOB_GEN_ENSEMBLE

```diff
@@ -18,7 +18,4 @@ if   [ ! -d RESTART ]; then mkdir -p RESTART; fi
 
 # Generate the ensemble using soca_enspert
 
-srun -n ${NPE} ${SOCA_EXEC}/soca_enspert.x ./yaml/genenspert.yml 
-
-#if   [ ! -d RESTART ]; then rm -rf RESTART; fi
-
+srun -n $npe_psoca ${SOCA_EXEC}/soca_enspert.x ./yaml/genenspert.yml 
```

Comment:
The npe_psoca was remaining from the previous version, the correct is NPE, as everything else that uses the soca.

Reply:
The $npe_psoca has been replaced with $NPE